src/Data/API/Value.hs

```diff
@@ -310,7 +311,7 @@ matchesNormAPIBasic bt v p = case (bt, v) of
     (BTint, Int _)       -> return ()
     (BTint, _)           -> Left (JSONError (expectedInt js_v), p)
     (BTutc, UTCTime _)   -> return ()
-    (BTutc, _)           -> Left (JSONError (Expected ExpString "UTCTime" js_v), p)
+    (BTutc, _)           -> Left (JSONError (Expected ExpString "UTCTime" js_v), p)  -- TODO: no @ExpUTCTime@ defined elsewhere, so error not accurate
```

Comment:
I wouldn't worry too much about precision of these errors, I think this is fine.